src/pages/ProductDetail/ProductDetail.scss

```
@@ -0,0 +1,178 @@
+@import url('https://fonts.googleapis.com/css2?family=Noto+Sans+KR:wght@300&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap');
+
+// font-family: 'Roboto', sans-serif;
+// font-family: 'Noto Sans KR', sans-serif;
+
+.productDetail {
+  width: 1200px;
+  font-family: 'Noto Sans KR', sans-serif;
+  margin: 0 auto;
+
+  .product {
+    display: flex;
+    justify-content: center;
+    font-size: 22px;
+    p {
+      margin: 6px 0px;
+    }
+    .productLeft {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      width: 300px;
+      height: 500px;
+      font-size: 30px;
+      .productName {
+        font-family: 'Roboto', sans-serif;
+      }
+      .price {
+        font-size: 15px;
```

```suggestion
      font-size: 30px;
      
      .productName {
        font-family: 'Roboto', sans-serif;
      }
      
      .price {
        font-size: 15px;
```
스타일 선택자 끼리는 한칸씩 띄워주세요!